test/util/labelledmatrix.jl

```diff
@@ -0,0 +1,131 @@
+module TestLSM
+
+if VERSION >= v"0.5-"
+    using Base.Test
+else
+    using BaseTestNext
+    const Test = BaseTestNext
+end
+
+import Bio.Util: readlsm
+
+integral_matrix = """\
+\tA\tB\tC
+A\t1\t2\t4
+B\t2\t1\t2
+C\t4\t2\t1
+"""
```

Comment:
Put these global variables inside a `@testset` macro. I think values used in a test should be placed near the test.